src/components/Person/Person.jsx

```diff
@@ -1 +1,21 @@
-// export const Person = ({ person }) => ();
+const PersonPartner = ({ person }) => (
+  <p className="Person__partner">
+    {person.isMarried ? (`${person.partnerName} is my ${person.sex === 'm' ? 'wife'
```

Comment:
using nested ternary operator is a bad practice, it's better to move it to the variable

Reply:
Why is it a bad practice? I maded another variable

Reply:
it's bad for reading and scalabling

Reply:
Will you remember to look at my code, I fixed the comment.